test_scripts/Polices/build_options/011_ATF_P_TC_PTS_Define_URL_to_send_PTS_HTTP.lua

```diff
@@ -94,26 +94,38 @@ end
 commonFunctions:newTestCasesGroup("Preconditions")
 
 function Test:ValidatePTS()
-  if ptu.policy_table.consumer_friendly_messages.messages then
-    self:FailTestCase("Expected absence of 'consumer_friendly_messages.messages' section in PTS")
+  if(ptu == nil) then
```

Comment:
@istoimenova, Currently there is an issue with ATF if 2 events with the same name are registered with Pin() option. Issue happens sometimes and leads to instability in scripts.
Please get rid of Pin() in lines 84 - 92